src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -855,7 +856,7 @@ def __export_keyvalues(fetched_items, format_, separator, prefix=None):
 
         if exported_dict and exported_list:
             logger.error("Can not export to a valid file! Some keys have been dropped. %s", json.dumps(
-                exported_dict, indent=2))
+                exported_dict, indent=2, ensure_ascii=False))
```

Comment:
Can you please remove this one since I'm changing it in my PR?

Reply:
sure thing